census/census.ipynb

```diff
@@ -32,16 +31,18 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%time\n",
-    "df = spatial.read_parquet('./data/census2010.parq')\n",
-    "df = df.persist()"
+    "df = dd.read_parquet('./data/census2010.parq/', engine='fastparquet').persist()\n",
```

Comment:
```suggestion
    %%time
    "df = dd.read_parquet('./data/census2010.parq/', engine='fastparquet').persist()\n",
```

As a general rule, I put `%%time` on any cell that takes more than a few seconds to run, or that a user might _wonder_ whether it takes more than a few seconds to run. Here, reading the file is usually only a few seconds, but if it were CSVs it would take minutes, so I think it's important to convey to someone reading the notebook when rendered on the website that this step should take e.g. 8 seconds.  They might think it would take 1 second or 1 minute, so showing them explicitly what it takes is helpful.